src/content/docs/release-notes/agent-release-notes/php-release-notes/php-agent-9190309.mdx

```diff
@@ -0,0 +1,27 @@
+---
+subject: PHP agent
+releaseDate: '2022-03-02'
+version: 9.19.0.309
```

Comment:
Idk if we have a section for "version:"
If we do, keep this as is. If not, add "version 9.19....." to the title

Reply:
The subject and version are combined to create page's title. Please have a look at how release notes look like in [preview](https://docswebsitedevelop-misiekndocswebsitephpagentrele.gtsb.io/docs/release-notes/agent-release-notes/php-release-notes/php-agent-9190309/).